### DISCUSSION TITLE
Delete photos in qfieldcloud

### DISCUSSION THREAD

**Discussion by megis-at** [opengisch/QField#2462] [Q&A]
I took some photos and synced them to the PC. How do I delete these photos(JPEG files) in the cloud? 
Best regards - Enrico

**Comment by marioba** [comment]
In order to provide a clearer and more manageable discussion forum and to have all discussions about QField and related products (i.e. QFieldCloud and QFieldSync) in a single place, we transferred all the discussions from the QFieldCloud to the [QField repository](https://github.com/opengisch/QField/discussions).

**Comment by suricactus** [comment]
@megis-at You can delete them using QFieldSync. First delete these photos from you local project on your PC. Then click synchronize on QFieldSync and select the "Local action" for these files. It will suggest to delete them on the cloud.

![image](https://user-images.githubusercontent.com/2820439/157393082-acdf0f2a-f24b-46ba-9c54-daed885f2966.png)

**Reply by suricactus** [reply]
Note in the future this will be possible also using the web UI, but this is what you can do for now.

**Comment by megis-at** [comment]
Ivan.

Thx for your mail.

Works fine!

Best regards
Enrico

Von: Ivan Ivanov ***@***.***>
Gesendet: Mittwoch, 9. März 2022 08:29
An: opengisch/QField ***@***.***>
Cc: Enrico Mahl ***@***.***>; Mention ***@***.***>
Betreff: Re: [opengisch/QField] Delete photos in qfieldcloud (Discussion #2462)


@megis-at<https://github.com/megis-at> You can delete them using QFieldSync. First delete these photos from you local project on your PC. Then click synchronize on QFieldSync and select the "Local action" for these files. It will suggest to delete them on the cloud.

[image]<https://user-images.githubusercontent.com/2820439/157393082-acdf0f2a-f24b-46ba-9c54-daed885f2966.png>

—
Reply to this email directly, view it on GitHub<https://github.com/opengisch/QField/discussions/2462#discussioncomment-2321993>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AREBUZID4PX3RCWDTFIJMOLU7BHKTANCNFSM5NJESZQA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
